fbw-a380x/src/wasm/systems/a380_systems/src/fuel/fuel_quantity_management_system.rs

```diff
@@ -177,13 +239,17 @@ impl SimulationElement for IntegratedRefuelPanel {
     }
 }
 
+const TRIM_TANK_TOML: &str = include_str!("./trim_tank_targets.toml");
```

Comment:
I really like the idea but this will increase the wasm size significantly. But I don't see another way either. Just something to keep in mind.

Maybe https://crates.io/crates/static-toml/ can be used instead?

Reply:
Probably for a future PR I think.